.github/workflows/spelling2.yml

```diff
@@ -43,8 +66,6 @@ on:
   pull_request_target:
     branches:
     - "**"
-    tags-ignore:
-    - "**"
```

Comment:
this was never supported by `pull_request_target` as it / `pull_request` never applied to tags.